### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by semratufan** [TechmongersNL/lighthouse#11]
The report is about the website https://www.dutchnews.nl/  which  written by Semra Tufan  on Tuesday 24-10-2023 at 161:15.
Buttons do not have an accessible name. When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. 

![image](https://github.com/TechmongersNL/lighthouse/assets/120501372/829ebfff-70d3-4d37-9ca1-1f16536ef554)

div.ms-fixed-row > div#ms-fixed-footer-container > div#ms-fixed-footer-spacer > button#ms-fixed-footer-close-btn
<button id="ms-fixed-footer-close-btn" class="msff-border msfr-clickable">
 
Here is the link for correct markup solutions.
https://dequeuniversity.com/rules/axe/4.7/button-name
